<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportPreferencesDialog</class>
 <widget class="QDialog" name="ReportPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="prefsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Page Format</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Paper Size:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="paperSizeComboBox">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>A0 [841 x 1189] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A1 [594 x 841] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A2 [420 x 594] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A3 [297 x 420] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A4 [210 x 297] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A5 [148 x 210] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A6 [105 x 148] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A7 [74 x 105] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A8 [52 x 74] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A9 [37 x 52] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B0 [1030 x 1456] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1 [728 x 1030] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2 [515 x 728] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B3 [364 x 515] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B4 [257 x 364] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B5 [182 x 257] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B6 [128 x 182] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B7 [91 x 128] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B8 [64 x 91] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B9 [45 x 64] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B10 [32 x 45] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C5E [163 x 229] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Comm10E [105 x 241] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DLE [110 x 220] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Executive [191 x 254] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Folio [210 x 330] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ledger [432 x 279] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Legal  [216 x 356] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Letter [216 x 279] mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tabloid [279 x 432] mm</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="portraitButton">
            <property name="text">
             <string>Portrait</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="lanscapeButton">
            <property name="text">
             <string>Landscape</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Margins</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Left:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="leftMarginSpinBox">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Top:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="topMarginSpinBox">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Right:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="rightMarginSpinBox">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Bottom:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="bottomMarginSpinBox">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Default Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="changeDefaultFontButton">
              <property name="text">
               <string>Change</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="defaultFontSampleTextEdit">
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.
Default Font Sample. Default Font Sample.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Header</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Spacing:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="headerRulerWitdhSpinBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Ruler color:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="headerHeightSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <double>60.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="headerSpacingSpinBox">
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Ruler width:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Height:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QToolButton" name="headerRulerColorButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>62</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Footer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="footerSpacingSpinBox">
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="footerRulerWitdhSpinBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Ruler color:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Height:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="footerHeightSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <double>60.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Ruler width:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QToolButton" name="footerRulerColorButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Spacing:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>62</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
